packages/common/interfaces/nest-application.interface.ts

```diff
@@ -48,24 +53,25 @@ export interface INestApplication extends INestApplicationContext {
   listenAsync(port: number | string, hostname?: string): Promise<any>;
 
   /**
-   * Registers the prefix for the every HTTP route path
+   * Registers a prefix for every HTTP route path.
    *
-   * @param  {string} prefix The prefix for the every HTTP route path (for example `/v1/api`)
+   * @param  {string} prefix The prefix for every HTTP route path (for example `/v1/api`)
    * @returns {void}
    */
   setGlobalPrefix(prefix: string): this;
 
   /**
    * Setup Ws Adapter which will be used inside Gateways.
-   * Use, when you want to override default `socket.io` library.
+   * Use when you want to override default `socket.io` library.
```

Comment:
```suggestion
   * Use when you want to override default `socket.io` library.
```

Could be anything - lets not assume it is socket.io